src/app/containers/Timestamp/relativeTimestamp.test.js

```diff
@@ -0,0 +1,86 @@
+import relativeTimestamp from './relativeTimestamp';
+
+const timeStampGenerator = milliseconds =>
+  Date.now() - milliseconds.timeDifference * milliseconds.magnitude;
+
+const timestampWithDiffInHours = hours =>
+  timeStampGenerator({
+    timeDifference: hours,
+    magnitude: 60 * 60 * 1000,
+  });
+
+const timestampWithDiffInMins = mins =>
+  timeStampGenerator({
+    timeDifference: mins,
+    magnitude: 60 * 1000,
+  });
+
+const relativeBehaviour = (description, input, expectedOutput) => {
+  it(description, () => {
+    const result = relativeTimestamp(input);
+    expect(result).toEqual(expectedOutput);
+  });
+};
+
+describe('relativeTimestamp', () => {
+  it('returns a string which ends in ago', () => {
+    const timestamp = timestampWithDiffInMins(1);
+    const result = relativeTimestamp(timestamp);
+    expect(typeof result).toEqual('string');
+    expect(result.split(' ').pop()).toEqual('ago');
+  });
+
+  relativeBehaviour(
+    'returns 1 minute ago',
+    timestampWithDiffInMins(1),
+    '1 minute ago',
+  );
+
+  relativeBehaviour(
+    'returns 5 minutes ago',
+    timestampWithDiffInMins(5),
+    '5 minutes ago',
+  );
+
+  relativeBehaviour(
+    'returns 1 hour ago',
+    timestampWithDiffInHours(1),
+    '1 hour ago',
+  );
+
+  relativeBehaviour(
+    'returns 5 hours ago',
+    timestampWithDiffInHours(5),
+    '5 hours ago',
+  );
+
+  relativeBehaviour(
+    'returns 5 hours ago',
+    timestampWithDiffInHours(5),
+    '5 hours ago',
+  );
+
+  relativeBehaviour(
+    'returns 1 minute ago for 2 milliseconds',
```

Comment:
Bug bare but instead of milliseconds can it be seconds? Also perhaps just use the same number of seconds for `returns 1 minute ago for 2 milliseconds` and `returns an empty string for 60 milliseconds in the future`

I think '10 seconds past and future' is clearer than 2 different amount of milliseconds which are arbitrary

Reply:
and done :)